docs/self-managed/setup/deploy/amazon/amazon-eks/eksctl.md

```diff
@@ -291,148 +298,256 @@ For detailed examples, review the [documentation provided by AWS](https://docs.a
   </p>
 </details>
 
-## PostgreSQL database
+## PostgreSQL Database
```

Comment:
afaik doc titles are always uppercase followed by lowercase, except it's a proper noun.